<launch>
    <!-- 启动 min_rects_processor_node 节点 -->
    <node name="min_rects_processor_node" pkg="choose_powerline" type="min_rects_processor_node" output="screen"/>
</launch>
